doc/relations-handling.md

```diff
@@ -15,10 +15,10 @@ Nelmio\Entity\User:
 Nelmio\Entity\Group:
     group1:
         name: Admins
-        owner: @user1
+        owner: '@user1'
 ```
 
-Alice also allows you to directly reference objects' properties using the ```@name->property``` notation.
+Alice also allows you to directly reference objects' properties using the ```'@name->property'``` notation.
```

Comment:
Sorry but those at line 7, 35 and here should not be quoted IMO, as they are not in the Yaml. The quotes aren't part of the alice syntax.